server/form-pages/apply/risks-and-needs/risk-of-serious-harm/additionalRiskInformation.ts

```diff
@@ -11,6 +11,8 @@ type AdditionalRiskInformationBody = Record<string, never>
   bodyProperties: [],
 })
 export default class AdditionalRiskInformation implements TaskListPage {
+  documentTitle = 'Additional risk information for the person'
```

Comment:
It seems like almost all of the example titles sub out the person's name for 'the person' - could we make a util function to generate both of them at the same time?

Reply:
I tooled around with a utility function, or something like
``` 
documentTitle = "Import the person's risk to self data from OASys"

title = this.documentTitle.replace('the person', this.personName)
```
but everything I tried just seemed over-complicated or less readable than just writing the string.. would be interested in any suggestions

Reply:
I quite like the utility function idea because to me it flags that we are deliberately setting these values differently, but I see what you're saying. It could be something like this:

```
function generateDocumentAndPageTitles(documentTitle: string, name: string) {
  const pageTitle = documentTitle.replace('the person', this.personName)

  return { documentTitle, pageTitle }
}
```

Reply:
ooh I also figured out how to add documentation to a field in typescript so that will add some immediate context so future devs won't have to look in git commits